transform_aes.c

```diff
@@ -38,6 +38,20 @@
 #define TRANSOP_AES_IV_KEY_BYTES (AES128_KEY_BYTES) /* use AES128 for IV encryption */
 #define TRANSOP_AES_PREAMBLE_SIZE (TRANSOP_AES_VER_SIZE + TRANSOP_AES_IV_SEED_SIZE)
 
+/* AES ciphertext preamble */
+#define TRANSOP_AES_COMPRESSION_SIZE   1
```

Comment:
I think this should be set to 0 when compression is disabled

Reply:
If we want to keep it as "encrypted flags field" instead, it should remain (unused) in the packet. It would allow compatibility among versions with and without encryption enabled. Let me know.

Reply:
As renamed now, as well defined generic field (8-bit size as discussed earlier), it could indicate other compression methods or some error correction codes or ... any other special data treatment option. 

I guess this is a design decision - do we want to permanently add this data flag field (for compatibility)?

Reply:
Done.